### DISCUSSION TITLE
Taskbar Image Missing (Windows 10 Education)

### DISCUSSION THREAD

**Discussion by MadtownLems** [iterate-ch/cyberduck#16620] [Q&A]
**Describe the bug**
Cyberduck's appearance in my taskbar is just the default broken image icon.

**Expected behavior**
A cute little duck

**Screenshots**
![2024-11-27 15_38_05-](https://github.com/user-attachments/assets/5191ef44-5fbe-47b1-b367-9d302e09446e)

**Desktop (please complete the following information):**
Windows 10 Education
Cyberduck 9.0.3

**Comment by Ankush1oo8** [comment]
Can you tell me how can fix it means I don't know in which file I can't fix this

**Comment by AliveDevil** [comment]
Interesting. Cyberduck Store, or standalone/Exe installed?
Which Education version? Check Settings [System/About](ms-settings:about).

**Comment by MadtownLems** [comment]
> Cyberduck Store, or standalone/Exe installed?

The cyberduck-installer.exe for windows from here: https://cyberduck.io/download/

> Which Education version? Check Settings System/About.

Version: 22H2
OS build: 19045.5131

**Comment by AliveDevil** [comment]
Can't seem to replicate here.
![image](https://github.com/user-attachments/assets/5f351f90-93ce-40e7-a0d8-74035c08855d)

Just installed both, Windows 10 and Cyberduck.
Did you use any software to optimize/cleanup your system?

**Comment by MadtownLems** [comment]
> Did you use any software to optimize/cleanup your system?

No - but it is a "managed" device by our campus IT, so I'm guessing that's it. Any suggestions on things I can look at to help debug? (confirm the image file exists where it should, for example)

Otherwise, it sounds like a "me" problem and don't stress it. Thanks for your time!

**Reply by AliveDevil** [reply]
One thing that comes to mind is that the Photos-app, or Windows Photo Viewer aren't active/removed, so that Windows can't decode the icon, but that would be really far-fetched.

The icon you should see is embedded as part of the Cyberduck executable (i.e. when navigating to "C:\Program Files\Cyberduck", the Cyberduck.exe should have the duck icon).

**Comment by MadtownLems** [comment]
C:\Program Files\Cyberduck shows the icon.

And I do have Photos installed and active.

Oh well. It doesn't impact functionality. I really do appreciate your time, with this and the application in general. Cheers!

**Reply by AliveDevil** [reply]
One thing you could do, if you are familiar with the command line:
Open a PowerShell terminal in `%LocalAppdata%\Microsoft\Windows\Explorer`.
Run command `Remove-Item -Include iconcache_*,thumbcache_* * -WhatIf`, verify that nothing else except files starting with iconcache and thumbcache are listed.
Kill Explorer.exe, keep TaskManager opened
Run the above command _without_ the `-WhatIf`, i.e. `Remove-Item -Include iconcache_*,thumbcache_* *`
In TaskManager start a new process "explorer.exe".
This should wipe the entire icon cache and rebuild it for you.
It may be a corrupt icon cache, and worth a shot.

**Comment by MadtownLems** [comment]
Good news! Today's Cyberduck update has restored the ducky in the taskbar! 🦆

![2024-12-06 10_49_20-](https://github.com/user-attachments/assets/475615cb-c837-4e0b-bb2f-d8b29338c9f6)
